.github/workflows/_check_app_params.yml

```diff
@@ -0,0 +1,41 @@
+name: Check APP_LOAD_PARAMS
+
+on:
+  workflow_call:
+    inputs:
+      ledger-app-workflows_ref:
+        description: 'The current reference in use for the ledger-app-workflow repository'
+        required: true
+        type: string
+      download_manifest_artifact_name:
+        description: 'The name of the artifact containing the built manifest'
+        required: true
+        type: string
+
+jobs:
+  check_app_paramss:
+    name: Check APP_LOAD_PARAMS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone workflows repository
+        uses: actions/checkout@v3
+        with:
+          repository: LedgerHQ/ledger-app-workflows
+          path: ./ledger-app-workflows
+          ref: ${{ inputs.ledger-app-workflows_ref }}
+
+      - name: Download manifest
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.download_manifest_artifact_name }}
+          path: ${{ inputs.download_manifest_artifact_name }}
+
+      - name: Download app params database
+        run: curl https://raw.githubusercontent.com/LedgerHQ/ledger-app-workflows/xch/app-params-check/app-params-database.json > app-params-database.json
```

Comment:
change to master? or V1? What do we prefer to be able to run against the up tot date database easily?